dynawo/sources/Modeler/DataInterface/PowSyblIIDM/DYNInjectorInterfaceIIDM.h

```diff
@@ -0,0 +1,132 @@
+//
+// Copyright (c) 2015-2020, RTE (http://www.rte-france.com)
+// See AUTHORS.txt
+// All rights reserved.
+// This Source Code Form is subject to the terms of the Mozilla Public
+// License, v. 2.0. If a copy of the MPL was not distributed with this
+// file, you can obtain one at http://mozilla.org/MPL/2.0/.
+// SPDX-License-Identifier: MPL-2.0
+//
+// This file is part of Dynawo, an hybrid C++/Modelica open source time domain
+// simulation tool for power systems.
+//
+
+//======================================================================
+/**
+ * @file  DYNInjectorInterfaceIIDM.h
+ *
+ * @brief Injector interface : header file for IIDM interface
+ *
+ */
+//======================================================================
+
+#include "DYNBusInterface.h"
+#include "DYNVoltageLevelInterface.h"
+
+#ifndef MODELER_DATAINTERFACE_POWSYBLIIDM_DYNINJECTORINTERFACEIIDM_H_
+#define MODELER_DATAINTERFACE_POWSYBLIIDM_DYNINJECTORINTERFACEIIDM_H_
+
+#include <powsybl/iidm/Injection.hpp>
+
+#include <boost/shared_ptr.hpp>
+#include <boost/weak_ptr.hpp>
+
+namespace DYN {
+
+/**
+ * class InjectorInterfaceIIDM
+ */
+template <class T>
```

Comment:
Useless template

Reply:
I agree. Can I make it a base-class ? It will have a test program then, and be more simple. May I wait until next class with injector to do the change ?

Reply:
We need to have a clean base class before moving to more complex models so I would prefer to do this as soon as possible. Regarding the test we can also test it through the child classes, I don't have a preference

Reply:
That's OK. I start right now

Reply:
done